### PR TITLE
Fetch ClusterTask without namespace if taskRef is ClusterTask

### PR DESCRIPTION
**Pull request by rohitkrai03**
Related Bug - https://jira.coreos.com/browse/ODC-2245

This PR -
- Updates `PipelineVisualizationTask` component to fetch `ClusterTask` if `taskRef` has a `kind: ClusterTask`.

### CODE REVIEW

**Comment by spadgett** [comment]
> could not copy stable imagestreamtag: Timeout: request did not complete within allowed duration

/retest

File: frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.tsx [hunk 1]
```diff
@@ -53,22 +54,33 @@ export const PipelineVisualizationTask: React.FC<PipelineVisualizationTaskProp>
...
       task.status &&
-      (task.status.reason !== runStatus.Succeeded && task.status.reason !== runStatus.Failed)
+      task.status.reason !== runStatus.Succeeded &&
+      task.status.reason !== runStatus.Failed
     ) {
       taskStatus.reason = runStatus.Cancelled;
     }
   }
+  let resources;
+  if (task.taskRef.kind === 'ClusterTask') {
```
Comment (gijohn) [inline frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.tsx]:
```suggestion
  if (task.taskRef.kind === ClusterTaskModel.kind) {
```

Comment (rohitkrai03) [reply frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.tsx]: Done.

**Pull request merged**